less/navigation.less

```diff
@@ -1 +1,56 @@
-// Navigation Styles here
\ No newline at end of file
+// Navigation Styles here
+
+header {
+    display: flex;
+    justify-content:center;
+    border-bottom: 2px dashed silver;
+    padding: 10px 0;
+    width: 100%;
+    align-items: center;
+
+    
+
+    .headerinfo {
+        width: 800px;
+        display: flex;
+        justify-content: space-between;
```

Comment:
your using display flex and justify content alot around here, it be good to make a mixin named flex that takes in the value of justify content to make it easy and fast to give a container display flex.